relay-general/src/store/normalize.rs

```diff
@@ -26,6 +26,7 @@ mod contexts;
 mod logentry;
 mod mechanism;
 mod request;
+pub mod spans;
```

Comment:
This is only a minor thought: Originally, the normalize module was thought of as rather opaque. Which of its contents do we need to expose? We might be able to find a better place for them. 

It looks like this is just the `SpanAttribute` enum.